### ISSUE TITLE
wiiurpxtool is generating broken rpx files.

### ISSUE THREAD

**Issue by Satoshi64** [BullyWiiPlaza/RPL-Studio#1]
Unpack a vc vessel.rpx and repack it again with the tool, console freezes.

**Comment by BullyWiiPlaza** [comment]
wiiurpxtool is being developed by this guy so posting there might be better:
https://github.com/0CBH0/wiiurpxtool

**Comment by Satoshi64** [comment]
I thought but seems dead because this bug is almost a year old lol

**Comment by Satoshi64** [comment]
Closing it.
If somebody else has this bug: The original rpx has value 04A0 at 0x462 and wiiurpxtool writes 04C0 on the new file. After restoring the value back to A0 with a hex editor the rpx works again.

**Comment by BullyWiiPlaza** [comment]
I can see if this can be fixed if you send me the `RPX` file in a private message on [`GBATemp`](http://gbatemp.net/members/352259) or something. Thanks.

**Comment by Satoshi64** [comment]
Sent!

**Comment by ShadowOne333** [comment]
Sorry to bump this.

I can confirm what Satoshi says.
Any re-compressed RPX file gives a black screen on boot.

However, the fix that Satoshi posted seems to work just fine.

The byte at hand is located exactly at 0x0463.
Recompressed RPX files with this tool changes this value to a C0.
Original RPX files have a value of A0 in here.

Hanging the value at 0x0463 from C0 to A0 makes the RPX work again.
Hope this helps to get the fix included into the JAR executable, I was struggling a lot today because I kept getting black screens until I saw the A0 in a Hex editor and changed it manually.
